views/templates/hook/onepagecheckout-p4x.tpl

```diff
@@ -61,7 +62,7 @@
 
         if (!checkSimulatorContent()) {
             sdk.simulator.init({
-                type: {$PAGANTIS_SIMULATOR_DISPLAY_TYPE_CHECKOUT|escape:'javascript':'UTF-8'},
+                type: '{$PAGANTIS_SIMULATOR_DISPLAY_TYPE_CHECKOUT|escape:'javascript':'UTF-8'}',
```

Comment:
This is not correct, this value must be a reference to the SDK, not a string. sdk.simulator.types.CHECKOUT_PAGE